config/locales/en.yml

```diff
@@ -1132,6 +1133,21 @@ en:
   ticket_column_unit_price: "Unit Price"
   ticket_column_total_price: "Total Price"
 
+  menu_1_title: "Shops"
+  menu_1_url: "/shops"
+  menu_2_title: "Map"
+  menu_2_url: "/map"
+  menu_3_title: "Producers"
+  menu_3_url: "/producers"
+  menu_4_title: "Connect"
+  menu_4_url: "https://openfoodnetwork.org/au/connect/"
+  menu_5_title: "Learn"
+  menu_5_url: "https://openfoodnetwork.org/au/learn/"
```

Comment:
I don't think these Aus links should be in the default settings for the base `en` locale. Hmmm... is the issue that Aus uses `en` without a separate language file?

Reply:
yes, as far as I understand, yes, this is the AUS config file :-)

Reply:
Yes, one downside of this approach is that whenever Aus wants to change the menu, we need to change `en.yml` and the changes will be available for translation again. That will probably be confusing for translators.